contracts/OtokenFactory.sol

```diff
@@ -0,0 +1,127 @@
+pragma solidity =0.6.0;
+
+import "./lib/Spawner.sol";
+import "./Otoken.sol";
+
+contract OtokenFactory is Spawner {
+    Otoken public logic;
+
+    address[] public otokens;
+    uint256 public otokensCreated;
```

Comment:
what we need this variable for ?

Reply:
`otokensCreated` is similar to `otokensCreated.length`, right ?

Reply:
Yes but you cannot get otokens.length off-chain

Reply:
That's why I think we should just make otokens private, and write our own public getter, that way we can just return the whole array, and we don't need otokensCreated.

Reply:
Agree. Or we can just make a function that return the length of the array, with `otoken` public, and loop through the array off-chain to get all of it, similar to the current implementation in OptionsProtocol repo, dev branch.

But either way could work (y)

Reply:
I prefer the a public function that return the whole array at once. @aparnakr any thoughts?